packages/component/src/Styles/StyleSet/SuggestedAction.js

```diff
@@ -18,12 +19,14 @@ export default function createSuggestedActionStyle({
     paddingTop: paddingRegular,
 
     '& > button': {
+      alignItems: 'center',
+      borderRadius: suggestedActionBorderRadius,
+      display: 'flex',
       fontFamily: primaryFont,
       fontSize: 'inherit',
-      height: suggestedActionHeight,
+      minHeight: suggestedActionHeight,
```

Comment:
switch back to `height`?

Reply:
also should we allow the user to affect the image size directly? Not just the suggested action?

Reply:
nvm I see it below haha